### DISCUSSION TITLE
Different colors for different feeds post entries

### DISCUSSION THREAD

**Discussion by rezad1393** [FreshRSS/FreshRSS#5473] [Q&A]
**Is your feature request related to a problem? Please describe.**
right now all the post from all feeds have the color. so in a list of post in a catogories I dont see which is for which feed based on color.
and because sort of post right now is not good (no sort based on publication nor feeds) it gets confusing. specially on my android phone that feeds titles are hidden and only post titles are there.

**Describe the solution you’d like**
different color for post belonging to different feeds when more than one feeds is shown. 
though it may not be completely possible when showing more than 5 feeds.

**Describe alternatives you’ve considered**
A clear and concise description of any alternative solutions or features you’ve considered.

**Additional context**

**Comment by math-GH** [comment]
Are you familiar with CSS?
With the extension CustomCSS, it is possible to configure it as you wish. Each line has a `data-feed` attribute with the ID of the feed.
With something like `.flux[data-feed=123] ...... {background-color: lightpink}` you could colour each feed

**Reply by Alkarex** [reply]
And if not sufficient, that would be a job for an extension:
* https://github.com/FreshRSS/Extensions
* https://freshrss.github.io/FreshRSS/en/developers/03_Backend/05_Extensions.html

**Comment by rezad1393** [comment]
after some thought I think it is not possible to do it easily.

because using multiple colors cant be done with more than ,say, 3 feeds. more than that and it becomes confusing.

I retract my suggestion.
sorry for wasting your time.

**Reply by Frenzie** [reply]
I'll drop a link to one form of a colorful tabs extension which does something like that.

https://github.com/MurzNN/TST-Colored-tabs

**Reply by rezad1393** [reply]
what?
what does have to do with color of feeds in freshrss?
the one you shared colors the tab of firefox. I was asking for feed posts list in freshrss.
I dont think they are related.

**Reply by Frenzie** [reply]
It's an example of an algorithm that's presumably already gone through a fair bit of fine-tuning and optimization.
